deploy/org.mavlink.qgroundcontrol.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mavlink.qgroundcontrol</id>
+  
+  <name>QGroundControl</name>
+  <summary>UAS ground control station</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+    <control>gamepad</control>
+  </recommends>
+  
+  <description>
+    <p>
+      Intuitive and Powerful Ground Control Station for the MAVLink protocol.
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">org.mavlink.qgroundcontrol.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://docs.qgroundcontrol.com/assets/quickstart/ConnectedVehicle.jpg</image>
+    </screenshot>
+  </screenshots>
+..
```

Comment:
This two dots should be here ?

Reply:
Heck?! Let me correct this.

Reply:
Done. Like your reviews @patrickelectric - you have eagle eyes :stuck_out_tongue: